src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import todosFromServer from '../../api/todos.json';
+
 export const TodoList = () => (
-  <>TodoList markup</>
+  <>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings
```suggestion
  <ul>
```